packages/popover/src/Popover/Popover.tsx

```diff
@@ -122,11 +122,11 @@ export const Popover = forwardRef<HTMLDivElement, PopoverComponentProps>(
     };
     const rootProps = usePortal ? portalProps : {};
 
-    const { placeholderRef, referenceElement, referenceElDocumentPos } =
+    const { referenceElement, referenceElDocumentPos, setPlaceholderElement } =
       useReferenceElement(refEl, scrollContainer);
     const { contentNodeRef, setContentNode } = useContentNode();
 
-    const { context, floatingStyles, placement, refs } = useFloating({
+    const { context, elements, placement, refs, strategy, x, y } = useFloating({
```

Comment:
Were `x`, 'y', and `strategy` different from what was returned in `floatingStyles`?